app/src/main/java/org/smartregister/addo/presenter/FamilyFocusedMemberProfileActivityPresenter.java

```diff
@@ -114,13 +117,19 @@ public void refreshProfileTopSection(CommonPersonObjectClient client) {
 
     @Override
     public void onSubmitted(boolean successful) {
+        if (successful) {
+            viewReference.get().displayProgressBar(false);
+            Toast.makeText((Activity) this.getView(), "Submitted ...", Toast.LENGTH_SHORT).show();
+        } else {
+            viewReference.get().displayProgressBar(true);
```

Comment:
I think this might cause an infinite loading for cases where the condition is false because there is no further action when the submission returns false it will simply `continue` to show the progress bar

Reply:
True that, I tested it and it did that. Currently though even the progress bar does not show, so maybe I can remove the else part?